core/server/worker/src/main/java/alluxio/worker/block/AsyncCacheRequestManager.java

```diff
@@ -91,8 +91,12 @@ public void submitRequest(AsyncCacheRequest request) {
       mAsyncCacheExecutor.submit(() -> {
         boolean result = false;
         try {
+          boolean isSourceLocal = mLocalWorkerHostname.equals(request.getSourceHost());
```

Comment:
How does this distinguish if the block is coming from an alluxio worker, or the ufs?

It looks like it is checking the source of the request of with the local worker hostname, and I'm not sure how if it is UFS or not can be determined by that?

Reply:
discussed offline

Reply:
O, seems like this code already existed before, and was just moved up to this location.